### DISCUSSION TITLE
Hey i have a error "Cannot access Dispatch before initialization"

### DISCUSSION THREAD

**Discussion by eitai** [reduxjs/redux-toolkit#2520] [Q&A]
![image](https://user-images.githubusercontent.com/35004992/179397977-f979f926-7b3d-40ce-b3d1-f709dc1086d3.png)
![image](https://user-images.githubusercontent.com/35004992/179397996-6a3c7e18-96b1-4a98-8708-809585e120e0.png)
![image](https://user-images.githubusercontent.com/35004992/179398020-fad89e90-8438-4b5c-8b10-32d995b6dcc4.png)

any idea?

**Comment by markerikson** [comment]
You need to call the **`useDispatch()` hook**, like `const dispatch = useDispatch()`.

**Reply by eitai** [reply]
oh OMG, thanks a lot ! didn't notice that
